src/pharmpy/tools/covsearch/tool.py

```diff
@@ -222,7 +224,14 @@ def create_workflow(
     return Workflow(wb)
 
 
+def _store_input_model(context, model, results, max_eval):
+    # Create links to input model
+    context.store_input_model_entry(ModelEntry.create(model=model, modelfit_results=results))
+    return max_eval
```

Comment:
Why pipe `max_eval` through this function? Couldn't it go directly to `_start`?

Reply:
This was a workaround to avoid having multiple endpoint tasks, so this way, `_store_input_model` became a part of the workflow.